master/reference/policy-controller/configuration.md

```diff
@@ -47,20 +47,21 @@ It is recommended to use the following configuration for API access:
 ## The leader election container
 
 The leader election container is an optional sidecar container which performs leader election using the Kubernetes API.
-This ensures that only a single instance of the policy controller is ever active.  
+This ensures that only a single instance of the policy controller is ever active.
 
-The leader election container is only recommended when running the policy controller as a static pod in a multi-master deployment. 
+The leader election container is only recommended when running the policy controller as a static pod in a multi-master deployment.
 
 However, it is instead recommended to use a `ReplicaSet` with a single replica to ensure that one instance
 will always be running without need for leader election.
 
 ### Kubernetes API access
 
-The leader election container also needs Kubernetes API access, which can be configured through a `kubeconfig` file placed in 
+The leader election container also needs Kubernetes API access, which can be configured through a `kubeconfig` file placed in
 the root directory of the container. This can be done by mounting a file from the host, or using Kubernetes [ConfigMap resources](http://kubernetes.io/docs/user-guide/configmap/).
 
 ### Other configuration
 
 * `LOG_LEVEL`: Supports the standard Python log levels. e.g. `LOG_LEVEL=debug`, defaults to `info`
+* `NO_MATCH_ORDER`: The order used for "no match" policies which implement default behavior.  New policies should not be created with an order greater than this value or else they will be ignored.  Default: 2000
```

Comment:
Is "New" significant here?  I don't think so.  In other words, I believe that any existing policies with order > NO_MATCH_ORDER would also be ignored.

May I suggest:
```
* `NO_MATCH_ORDER`: The order used for the "no match" policies which implement default behavior.  Policies with an order greater than this value will be ignored.  Default: 2000
```